app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,253 @@
+/*
+ * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * listed below.
+ *
+ * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
+ * vendor/assets/stylesheets directory can be referenced here using a relative path.
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *
+ *= require_tree .
+ *= require_self
+ */
+
+$blue: #3b74b4;
+$white: #fff;
+$grey: #e4e4e4;
+$secondary-white: #f7f7f7;
+$green: #67b930;
+$normal-grey: #616161;
+$light-grey: #8d8d8d;
+$dark-grey: #2b2b2b;
+$red: #c72727;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  font-weight: 100;
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  min-width: 300px;
+}
```

Comment:
- [ ] Based on the template design provided for this project, the fonts to be used should be:

- **_Proxima Nova Light_** for general text and
- **_Proxima Nova Bold_** for Headers

Kindly use these fonts instead of the _**Poppins**_ font. 👍 😎  

![image](https://github.com/ishimwepati/Smart-Pocket-Budget-App/assets/77106962/b83aab1a-b3b6-4276-83a5-bc18240e41d3)

Reply:
Thanks for this : Done With It ! 

![code](https://github.com/ishimwepati/Smart-Pocket-Budget-App/assets/14818338/6f996fd6-ec0d-4f4e-aa33-ce842be865a8)

![code](https://github.com/ishimwepati/Smart-Pocket-Budget-App/assets/14818338/8b55f84b-b1ed-4271-a437-05f2431aefb0)

Reply:
Kindly not that it can't delay to appear on the deployment, But Locally it has worked!